pcsx2/CMakeLists.txt

```diff
@@ -912,6 +1008,7 @@ if(Linux)
 		
 	set(Platform_Libs
 		${LIBUDEV_LIBRARIES}
+        ${X11_LIBRARIES}
```

Comment:
I get that our cmake indentation is all over the place, but at least be consistent within one statement

Reply:
This isn't voluntary! I didn't double check my cmake indentation, will fix